Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino

```diff
@@ -136,7 +139,7 @@ void set_freq(){
   arg = SCmd.next();
   frequency = atof(arg);
   if (arg != NULL){
-    if(frequency > 902 && frequency < 923){
```

Comment:
This 923 frequency only applies for US915, but any user should be able to put any frequency they desire

Reply:
Frequency restriction removed. TODO: restrict frequency and channels according to region